superset-frontend/src/components/ProgressBar/index.tsx

```diff
@@ -30,13 +30,18 @@ const ProgressBar = styled(({ striped, ...props }: ProgressBarProps) => (
   <AntdProgress {...props} />
 ))`
   line-height: 0;
+  position: inherit;
+  .ant-progress-inner {
+    position: inherit;
+  }
   .ant-progress-outer {
     ${({ percent }) => !percent && `display: none;`}
   }
   .ant-progress-text {
     font-size: ${({ theme }) => theme.typography.sizes.s}px;
   }
   .ant-progress-bg {
+    position: inherit;
```

Comment:
curious, do you know specifically what this value should be? It would be better to be more explicit if you can.

Reply:
no, I don't but I can test it. I think it should probably be absolute?

Reply:
@eschutho it was static!

Reply:
great!